### PR TITLE
Added tooltip to auctions; Highlight active auction. Add label

### PR DESCRIPTION
**Pull request by AudioSin**
- Added tooltip to auctions
- Highlight active auction
- Add label for weekly auction

### CODE REVIEW

File: src/scss/_dark-tables.scss [hunk 1]
```diff
@@ -16,7 +16,7 @@
     }
   }
   tbody {
-    tr.weekly-auction {
+    tr.active {
```
Comment (Mvii7) [inline src/scss/_dark-tables.scss:19]: active is not a very explicit class name, I'd call it todays-auction

**Review by Mvii7 — CHANGES_REQUESTED** [review]
..

File: src/app/auction-page/auction-page.component.html [hunk 2]
```diff
@@ -335,8 +348,8 @@
         </tr>
       </thead>
       <tbody>
-        <tr *ngFor="let auction of auctions" [ngClass]="{'weekly-auction': auction.isWeekly}">
-          <td>{{ auction.time.date | date: "shortDate" }}</td>
+        <tr *ngFor="let auction of auctions" [ngClass]="{'weekly-auction': auction.isWeekly, 'active': auction.time.state === 'progress'}">
```
Comment (Mvii7) [inline src/app/auction-page/auction-page.component.html:351]: Remove weekly-auction bits pls from here and component

Comment (AudioSin) [reply src/app/auction-page/auction-page.component.html:351]: Why would I? I'm using it...

Comment (Mvii7) [reply src/app/auction-page/auction-page.component.html]: Yeah u are 👍

**Review by Mvii7 — APPROVED** [review]

**Pull request merged**